import.go

```diff
@@ -125,28 +224,25 @@ func (i *Importer) Add(exportNode *ExportNode) error {
 
 	buf := bufPool.Get().(*bytes.Buffer)
 	buf.Reset()
-	defer bufPool.Put(buf)
-
-	if err = node.writeBytes(buf); err != nil {
-		return err
+	if err := node.writeBytes(buf); err != nil {
+		panic(err)
```

Comment:
why `panic` here?

Reply:
ah good catch, let me fix it

Reply:
@cool-develope Should be fixed now! Please take another look, thanks!